ansibullbot/triagers/ansible.py

```diff
@@ -99,20 +99,12 @@
 from ansibullbot.parsers.botmetadata import BotMetadataParser
 
 
-REPOS = [
-    u'ansible-collections/community.general',
-]
-
-# Collection Repos, where Bot is allowed to run
-CREPOS = [
-    u'ansible-collections/community.general',
-    u'ansible-collections/aws',
-    u'ansible-collections/windows',
-]
-
+REPOS = C.DEFAULT_REPOS
 repo_data = {'ansible-collections/community.general': {'branch': 'master', 'shippable_prj' : '5e664a167c32620006c9fa50'},
-        }
-
+             'ansible-collections/community.aws': {'branch': 'main', 'shippable_prj' : '5e5ed2ae0fcc0d0006d2c037'},
```

Comment:
Ah, this is my fault, `shippable_prj` isn't used by anything, I couldn't work out how to pass this data structure into the Shippable code. This is why you are seeing `stale_ci`